astropy/coordinates/angles.py

```diff
@@ -190,7 +190,7 @@ def signed_dms(self):
         """
         The angle's value in degrees, as a named tuple with ``(sign, d, m, s)``
         members.  The ``d``, ``m``, ``s`` are thus always positive, and the sign of
-        the angle is given by ``sign``. (This is a read-only property.)
+        the angle is given by ``sign``. (This is a read-only property.).
```

Comment:
Same

Reply:
✅. I've removed "This is a read-only property" and also reformatted so there's a summary line.